docs/configuration/configuration-file.mdx

```diff
@@ -5,224 +5,453 @@ import { StyledContent } from '../../components/styled.elements';
 
 # Redocly configuration file
 
-The Redocly configuration file controls options for the OpenAPI CLI tool.
-It is also used across other Redocly products to help you control their behavior.
+The Redocly configuration file is used by OpenAPI CLI and other Redocly apps to control their behavior — from the strictness of the `lint` command, to how Redocly renders your docs.
 
-- Workflows uses it in the API registry to manage your APIs and control advanced features like region and link resolution.
-- Reference uses it to apply your custom settings when building API reference documentation (hosted and on-premise).
-- VS Code extension uses it for linting criteria, to apply custom settings for live documentation preview, and to set API definition root files.
+* Workflows uses it in the API registry to manage your APIs and control advanced features like region and link resolution.
+* Reference uses it to apply your custom settings when it's building API reference documentation (hosted and on-premise).
+* VS Code extension uses it for linting criteria, to apply custom settings for live documentation previews, and to set API definition root files.
 
-## Filename and location
+## Location, name and format
 
-The Redocly configuration file must be named `redocly.yaml` or `.redocly.yaml`, and placed into your working directory or the root of your project (depending on the Redocly product you're using).
+The Redocly configuration file must be named either `redocly.yaml` or `.redocly.yaml`. It must be located in the root of your project directory, and it must be in the YAML format. If you cloned the `openapi-starter` project, this is already set up for you.
 
-OpenAPI CLI will search for the configuration file in the working directory.
+All of the available sections and config options are documented below.
 
```

Comment:
Missing the top level object documentation.

Reply:
I'm not sure what you mean, sorry Adam.

Reply:
Maybe something like this? https://github.com/Redocly/openapi-cli/pull/666#discussion_r863979880

Reply:
I removed it because I thought it might be overkill. Clearly got that wrong. :) I'll reinstate it.